code/panels/UpdatePanel.php

```diff
@@ -83,28 +85,42 @@ public function getLatestSilverStripeVersion() {
 		return $latestVersion;
 	}
 
-	public function isCurrentSilverStripeVersion() {
-		if ($this->getCurrentSilverStripeVersion() && $this->getLatestSilverStripeVersion()) {
-			return $this->getCurrentSilverStripeVersion() >= $this->getLatestSilverStripeVersion();
+	public function isNewerSilverStripeVersion($versionNumber, $latestVersionNumber) {
+		if ($versionNumber == $latestVersionNumber) {
```

Comment:
Yeah, maybe `isNewestSilverStripeVersion` makes more sense - it checks whether the first value is the newest or the same as the second value.

I agree about keeping the logic there, it is a nice check-and-skip case.